Geometry_Engine/Query/Bounds.cs

```diff
@@ -416,9 +416,22 @@ public static BoundingBox Bounds(this CompositeGeometry group)
 
         public static BoundingBox IBounds(this IGeometry geometry)
         {
+            if (geometry == null)
+                return null;
+
             return Bounds(geometry as dynamic);
         }
 
+
+        /***************************************************/
+        /**** Private Methods - Fallback                ****/
+        /***************************************************/
+
+        private static BoundingBox Bounds(object value)
```

Comment:
Would probably switch this method to only accept `IGeometry` rather than `object`. Should not really make any real difference, but will look more sensible code-wise.

Reply:
@epignatelli could you please reflect on that?

Reply:
Yeah, I agree with @IsakNaslundBh - it doesn't change anything, it may be just more sensible.
I am pretty flexible about it - happy with both anyway.

Reply:
Updated.